backend/fms_core/templates.py

```diff
@@ -147,6 +148,27 @@
       ("Library", "Sample Container Coordinates", "coordinates"),],
 }
 
+LIBRARY_QC_TEMPLATE = {
+  "identity": {"description": "Template to add library QC", "file": static("submission_templates/Library_QC_v3_9_0.xlsx")},
```

Comment:
Small stuff, the front end uses a different icon if you include quality in the description. It would be better too to match the sample quality control description. Use something like : Template to perform library quality control

Reply:
Prefiling the template seem to be killing the drop downs. Try removing all content and formats from the "empty" lines between the Title and the header in the template. Hopefully this is going to fix it.